tests/integration_test.rs

```diff
@@ -6,7 +6,7 @@ use mkchlog::template::Template;
 use mocks::GitCmdMock;
 use std::fs::File;
 
-const YAML_FILE: &str = ".mkchlog.yml";
+const YAML_FILE: &str = "tests/mkchlog.yml";
```

Comment:
What is the reasoning behind this?
Main config file `.mkchlog.yml` which serves as a reference file for end users would be untested and could be possibly broken and we would have to maintain 2 files that ideally should have identical content. Plus when I implement issue #2 I definitely will write a test for it and would need to add the new settings key `skip-commits-up-to` to both files.

Reply:
It will be partially tested (`check`) by using it in this project. I don't think it makes sense to change the tests every time this project changes its changelog configuration. (The tests currently break if the current `.mkchlog.yml` is used.) Note that I found a bug that it displays Security section even when there are no security commits - that's a different issue.

We should point people at `tests/mkchlog.yml` as a reference file. (Or move it elsewhere.)

Didn't notice your comment so I already did #3.

Reply:
Ok, got it, thanks!